### PR TITLE
CSI-1492 remove svc cli result reader

### PR DESCRIPTION
**Pull request by oriyarde**
fix https://github.com/IBM/ibm-block-csi-driver/pull/154

### CODE REVIEW

File: controller/array_action/array_mediator_svc.py [hunk 1]
```diff
@@ -701,23 +698,30 @@ def delete_snapshot(self, snapshot_id):
...
                              "{1}".format(initiators.nvme_nqn, nvme_host))
-            if initiators.is_array_wwns_match(host.wwns):
+            host_wwns = self._get_host_ports(host, HOST_WWPNS_PARAM)
+            if initiators.is_array_wwns_match(host_wwns):
                 fc_host = host.name
                 connectivity_types.add(config.FC_CONNECTIVITY_TYPE)
                 logger.debug("found fc wwns in list : {0} for host : "
                              "{1}".format(initiators.fc_wwns, fc_host))
-            if initiators.is_array_iscsi_iqns_match(host.iscsi_names):
+            host_iqns = self._get_host_ports(host, HOST_ISCSI_NAMES_PARAM)
```
Comment (ArbelNathan) [inline controller/array_action/array_mediator_svc.py:723]: could the field `'iscsi_name'` contain a list? and why its called `ISCSI_NAMES` and not `ISCSI_NAME`?

Comment (oriyarde) [reply controller/array_action/array_mediator_svc.py:723]: yes, I assume because there can be more than 1 (same as `WWPNS`)

Comment (ArbelNathan) [reply controller/array_action/array_mediator_svc.py:723]: but its `PARAM`, the parameter name is `ISCSI_NAME`, `iscsi_names` would be the `value`..

Comment (ArbelNathan) [reply controller/array_action/array_mediator_svc.py:723]: what about `NQN`? should it be `NQNS`?

Comment (oriyarde) [reply controller/array_action/array_mediator_svc.py]: I agree the "number declension" is inconsistent, and none of them are parameters anyway (I didn't choose these names)

Comment (oriyarde) [reply controller/array_action/array_mediator_svc.py]: I aligned them. hope it's ok now

Comment (ArbelNathan) [reply controller/array_action/array_mediator_svc.py]: tnx

**Review by ArbelNathan — APPROVED** [review]
LGTM GJ!

**Pull request merged**